### ISSUE TITLE
Pinging hypixel

### ISSUE THREAD

**Issue by Joly0** [FragLand/minestat#77]
i am referencing #50 here kinda.

I am trying to tinker around with this and was curious why mc.hypixel.net on port 25565 always gives back "Server ist offline" although the server is not offline. I have read in the bug report above, that the java implementation is not capable of using the correct slp protocol, but according to another bug report, the c# version should support all slp protocols and therefore should be able to retrieve the information, shouldnt it?

**Comment by mindsolve** [comment]
Hi @Joly0!

Issue 50 regarding the support for the MC 1.7 (`json`) SLP protocol is fixed since the commit mentioned in the last comment there. You can find the current implementation status in issue #43. I have tested the Java and Python versions against a vanilla and a PaperMC server, which worked flawlessly.
There seems to be some kind of special behaviour regarding Hypixel.
I will have a look at this.

**Comment by mindsolve** [comment]
Hi @Joly0, thanks again for reporting this!

We found the problem.
The SLP protocol uses JSON as data format. The json data for Hypixel doesn't contain the MOTD in the same place as the vanilla and PaperMC servers do (`description.text`), which we used as the basis for our implementations.

For the Python version, PR #78 will resolve this problem, we hope to have a fix ready for the Java version soon.
